### PR TITLE
Update ChangeLog and README (SHOOP-987)

### PR DESCRIPTION
**Pull request by suutari-ai**
* ChangeLog: Add all changes after 1.0.0
* README: Remove text about DCO
* README: Change markdown title formatting

### CODE REVIEW

**Comment by suutari-ai** [comment]
Oh, snap.. wait.. I'll have to check the ChangeLog formatting. It does not look good when rendered through Github...

**Comment by suutari-ai** [comment]
The ChangeLog formatting should be OK after all. It's just Github styling that renders list items having subitems with uneven spacing. (See the spacing after "Remove Stripe integration" item in https://github.com/suutari-ai/shoop/blob/28d1d5a1eef5d4b14184666fae7b58df0996ab08/ChangeLog.md) I don't think there is much we can do about that.

**Comment by suutari-ai** [comment]
Oops.. I lost some changes..

File: README.md [hunk 1]
```diff
@@ -12,14 +14,15 @@ Shoop is International Registered Trademark & Property of Shoop Ltd.,
...
-## CLA & DCO
+CLA
+---
 
 Notice: At the release of Shoop Marketplace we will be adding
-Contributor License Agreement and Developer's Certificate of Origin to
-allow the best possible experience for users to sell add-ons thru
-Marketplace
+Contributor License Agreement for Shoop to allow the best possible
+experience for users to sell add-ons thru Shoop Marketplace.
```
Comment (akx) [inline README.md]: > through

Comment (suutari-ai) [inline README.md]: will fix that, thanks...

**Comment by suutari-ai** [comment]
Fixed: thru -> through.

**Pull request merged**